src/backend/distributed/citus--7.1-2--7.1-3.sql

```diff
@@ -0,0 +1,10 @@
+/* citus--7.1-2--7.1-3 */
+
+CREATE TABLE citus.pg_dist_metadata(
+    tag text NOT NULL,
+    value text NOT NULL
+);
+ALTER TABLE citus.pg_dist_metadata SET SCHEMA pg_catalog;
+GRANT SELECT ON pg_catalog.pg_dist_metadata TO public;
```

Comment:
By default, user who created the extension owns the table and has all the permissions. So I think we should be OK in docker.

We probably don't need this `GRANT` for `pg_dist_metadata` since we run it in a background worker. I just added it since other `pg_dist...` tables had this `GRANT` so all users can read these tables.